app/lib/account_session.rb

```diff
@@ -51,13 +51,45 @@ def serialise
 
   def to_hash
     {
-      access_token: access_token,
-      refresh_token: refresh_token,
+      access_token: @access_token,
+      refresh_token: @refresh_token,
       level_of_authentication: level_of_authentication,
     }
   end
 
+  def get_remote_attributes(remote_attributes)
+    remote_attributes.index_with { |name| oidc_do :get_attribute, { attribute: name } }
```

Comment:
I didn't know `index_with`, nice one!

Reply:
It's actually a rubocop suggestion, if your `foo.each_with_object({})` is sufficiently simple